src/components/Notifications.js

```diff
@@ -87,31 +61,7 @@ const Notifications = () => {
         const seenBy = notification.seenBy.map((seen) => seen.id)
         if (!seenBy.includes(userId)) {
           seenBy.push(userId)
-          await axios.post(
-            `${apiURL}/graphql`,
-            {
-              query: `mutation updateNotifications($seenBy: [ID]){
-                updateUserStoryNotification(input: {
-                  where: {
-                    id: "${notification.id}"
-                  }
-                  data: {
-                    seenBy: $seenBy
-                  }
-                }) {
-                  userStoryNotification {
-                    id
-                  }
-                }
-              }`,
-              variables: {
-                seenBy: seenBy
-              }
-            },
-            {
-              withCredentials: true
-            }
-          )
+          await userStory.markNotificationAsRead(notification.id)
```

Comment:
seenBy variable is not being passed to markNotificationAsRead

Reply:
Done now! Thanks :)